src/app/planner/planner/planner.component.html

```diff
@@ -291,17 +291,55 @@ <h4 class="modal-title pull-left">Create a Planner Event</h4>
 					<fieldset class="form-group">
 						<legend>Event Date</legend>
 						<div class="event-date btn-group">
-							<input
-								#drpCreate="bsDaterangepicker"
-								bsDaterangepicker
-								[bsConfig]="daterangeOptions"
-								placement="top"
-								class="form-control"
-								name="dates"
-								[(ngModel)]="createEventModel.dates">
-							<button class="btn btn-secondary" (click)="drpCreate.toggle()">
-								<fa-icon icon="calendar"></fa-icon>
-							</button>
+							<div class="form-group hidden">
+								<div class="input-group">
+								  <input name="datepicker"
+										 class="form-control"
+										 ngbDatepicker
+										 #datepicker="ngbDatepicker"
+										 [autoClose]="'outside'"
+										 (dateSelect)="onDateSelection($event)"
+										 [displayMonths]="2"
+										 [dayTemplate]="t"
+										 outsideDays="hidden"
+										 [startDate]="fromDate!"
+										 tabindex="-1">
+								  <ng-template #t let-date let-focused="focused">
+									<span class="custom-day"
+										  [class.focused]="focused"
+										  [class.range]="isRange(date)"
+										  [class.faded]="isHovered(date) || isInside(date)"
+										  (mouseenter)="hoveredDate = date"
+										  (mouseleave)="hoveredDate = null">
+									  {{ date.day }}
+									</span>
+								  </ng-template>
+								</div>
+							  </div>
+							  <div class="form-group">
+								<div class="input-group">
+								  <input #dpFromDate
+										 class="form-control" placeholder="yyyy-mm-dd"
+										 name="dpFromDate"
+										 [value]="formatter.format(fromDate)"
+										 (input)="fromDate = validateInput(fromDate, dpFromDate.value)">
+								  <div class="input-group-append">
+									<button class="btn btn-outline-secondary calendar" (click)="datepicker.toggle()" type="button"> <fa-icon icon="calendar"></fa-icon></button>
+								  </div>
+								</div>
+							  </div>
+							  <div class="form-group ml-2">
+								<div class="input-group">
+								  <input #dpToDate
+										 class="form-control" placeholder="yyyy-mm-dd"
+										 name="dpToDate"
+										 [value]="formatter.format(toDate)"
+										 (input)="toDate = validateInput(toDate, dpToDate.value)">
+								  <div class="input-group-append">
+									<button class="btn btn-outline-secondary calendar" (click)="datepicker.toggle()" type="button"> <fa-icon icon="calendar"></fa-icon></button>
+								  </div>
+								</div>
+							</div>
```

Comment:
I'm fine with this being a copypaste from the ngb docs but we should probably either mention that or put a TODO to clean it up later

Reply:
yeah that would be my plan: fixing this asap when I have the time.